app/models/practice.rb

```diff
@@ -70,7 +70,8 @@ def origin_picture_s3_presigned_url(style = nil)
   validates_attachment_content_type :main_display_image, content_type: /\Aimage\/.*\z/
   validates_attachment_content_type :origin_picture, content_type: /\Aimage\/.*\z/
   validates :name, presence: { message: 'Practice name can\'t be blank'}
-  validates :tagline, presence: { message: 'Practice tagline can\'t be blank'}
+  validates_uniqueness_of :name, { message: 'Practice name already exists'}
+  # validates :tagline, presence: { message: 'Practice tagline can\'t be blank'}
```

Comment:
@JoshingYou1 this may need to come back for the story you are working on, DM-1471.
Let me know/let's discuss.